### ISSUE TITLE
Why,Downgrade.

### ISSUE THREAD

**Issue by Stefan-comkmits** [mumble-voip/mumble#5816]
### The issue

Mumble was Complet OK.
Now,the Icons too Small,the custom Layoutu Dosent Work.

Why?
Has Discord Payt to make Kill the Dead?

Im sad,that Mumble not goas forward the Right way,
juts killing Mumble,fpor the last.

:(

### Mumble version

1.4.0

### Mumble component

Client

### OS

Windows

### Additional information

Mumble was Complet OK.
Now,the Icons too Small,the custom Layoutu Dosent Work.

Why?
Has Discord Payt to make Kill the Dead?

Im sad,that Mumble not goas forward the Right way,
juts killing Mumble,fpor the last.

:(

**Comment by ZeroAbility** [comment]
> Mumble was Complet OK.
Now,the Icons too Small,the custom Layoutu Dosent Work.

We will need some more specific information about what is "too small," and "doesn't work," before we can assist with this. I would recommend looking over this section as well as our documentation in general on our wiki:
https://wiki.mumble.info/wiki/Features#Customizability

Please be advised, we are in the process of migrating that to our website so the location of documentation will change in the future.

>Why?
Has Discord Payt to make Kill the Dead?

No, Discord does not pay us and we are not involved with them. I don't really see how that helps fix the UI, though.

Finally, your title doesn't really help other people find this issue and get helpful information to fix an issue that may be the same or similar as yours. You might want to change the title of this issue so that it is easier to find.

**Comment by Tha14** [comment]
![image](https://user-images.githubusercontent.com/5931740/186005333-b8d0271a-2ef5-4ec7-b62f-a96531daaff3.png)

@ZeroAbility Those icons were not this small before the last update. This is on windows 10, no DPI scaling is at 100%. Also, the chat text is much smaller than it used to be.

**Comment by Hartmnt** [comment]
@Tha14 

The icon issue is being tracked in #5817 and solved with #5822

**Comment by Hartmnt** [comment]
If you find separate issues unrelated to anything being tracked, please open a new issue

**Comment by Krzmbrzl** [comment]
Closing this due to lack of response
